src/reference/asciidoc/jdbc.adoc

```diff
@@ -575,6 +578,29 @@ The message data for a persistent channel is keyed in the store on the channel n
 Consequently, if the channel names are not globally unique, the channels can pick up data that is not intended for them.
 To avoid this danger, you can use the message store `region` to keep data separate for different physical channels that have the same logical name.
 
+==== PostgreSQL: receiving push notifications
```

Comment:
Add, please, an `id` to this section, e.g. `[[postgres-notifications]]`.
And then add a new section in the `whats-new.adoc` in the `New Components` paragraph.